src/table/column.rs

```diff
@@ -64,6 +64,7 @@ pub enum ColumnSpec {
     UniqueKey,
     PrimaryKey,
     Extra(String),
+    Check(String)
```

Comment:
Hey @LemarAb, thanks for contributing!!

We should represent the check expression in `SimpleExpr`.